generate_previews.py

```
@@ -0,0 +1,389 @@
+
+Schedule widget: day column (label + date number) | divider | items with bg rects
+Canteen widget:  day column (label + "d MMM")     | divider | items without bg rects
+
+Emoji characters are used for canteen meal types (monochrome Roboto glyphs).
+Price format matches NumberFormat.getCurrencyInstance(Locale.GERMANY): "3,60 €"
+"""
+
+from PIL import Image, ImageDraw, ImageFont
+
+# -- Fonts (Roboto, downloaded to /tmp/roboto/) -----------------------------
+FONT_REG = "/tmp/roboto/Roboto-Regular.ttf"
```

Ruff (0.15.4)

[error] Probable insecure usage of temporary file or directory: "/tmp/roboto/Roboto-Regular.ttf"

(S108)